api/src/main/java/org/apache/cloudstack/api/response/HostResponse.java

```diff
@@ -114,9 +114,9 @@
     @Param(description = "the amount of the host's CPU after applying the cpu.overprovisioning.factor ")
     private String cpuWithOverprovisioning;
 
-    @SerializedName("averageload")
+    @SerializedName("cpuloadaverage")
```

Comment:
Can you move this to APIConstants, also changes in the API key response breaks the response compatibility if xen-specific users are using the key for something in their scripts, tool, external integrations. Can you revert the change?

Reply:
It seems to me the guys with xen can't use it anyway:
https://github.com/apache/cloudstack/blob/1d05fead49f5c856257a741b07122f5633d2e359/plugins/hypervisors/xenserver/src/main/java/com/cloud/hypervisor/xenserver/resource/CitrixResourceBase.java#L2258-L2272

I will add it to the APIConstanst and leave the API key changed for consistency. Sounds good to you @rhtyd

Reply:
I'm okay with your rename change if the field is not set previously.

Reply:
Okay, I think [this](https://github.com/apache/cloudstack/pull/3738/commits/e05667692505989001764324a165831420628b05) should do it. I couldn't find any code where the old name is used